source/guides/put-it-together/ddiscordbot.md

```diff
@@ -0,0 +1,303 @@
+Denizen and Discord: dDiscordBot
+--------------------------------
+
+```eval_rst
+.. contents:: Table of Contents
+    :local:
+```
+
+### Introduction
+
+dDiscordBot is an **addon plugin** for Denizen that provides commands, script events, and tags for interacting with the Discord API. Basically, you can create a functioning Discord bot directly within Denizen!
+
+### What Can It Do?
+
+There are many libraries for various different programming languages for the Discord API. You might be familiar with some, such as [discord.js](https://discord.js.org), [discord.py](https://discordpy.readthedocs.io/en/stable/), and [JDA](https://github.com/DV8FromTheWorld/JDA). These libraries aim to cover **all aspects** of the API.
+
+Denizen is for Minecraft servers. Accordingly, dDiscordBot provides utilities that would be useful for a Minecraft server to interact with Discord. Nevertheless, some of the things you can do are:
+
+- Send, receive, and reply to messages
+  - Attach files, embeds, and buttons
+- Manage and reply to slash commands
+- Manage roles
+
+...and much more.
+
+### What Can It Be Used For?
+
+Many Minecraft servers have Discord communities. They provide a social space outside of the game itself. Discord users might want to interact with the online Minecraft players, and vice versa. 
+
+The main uses might be an account linker, which verifies a Discord user with their player on the server, and a chat bridge between Discord and Minecraft. You might also relay announcements, host cross-platform events, and let Discord users query data from the server.
+
+### Creating a Bot
+
+There are enough tutorials out there for creating a bot account. [Here's a handy one](https://discordpy.readthedocs.io/en/stable/discord.html).
+
+### Logging In
+
+To log in using dDiscordBot, use the `discordconnect` command. **Don't hardcode your token into the script!** Use a separate tokenfile placed outside of the `scripts` folder. A common location for this is in `Denizen/data`.
```

Comment:
The `discordconnect` command can't actually take the token as an argument like our old `discord connect` command could, would probably just write that you save the token in an external file to prevent it from staying in ram for security reasons